src/selectors/columnWidths.js

```diff
@@ -164,55 +175,18 @@ function flexWidths(columnGroupProps, columnProps, viewportWidth) {
  *   scrollableColumns: !Array.<columnDefinition>
  * }}
  */
-function groupElements(elements) {
```

Comment:
Besides grouping, `groupElements` also assigned `index` and `offset`.
The extra work is no longer needed.